apps/system/test/unit/utility_tray_test.js

```diff
@@ -603,6 +624,37 @@ suite('system/UtilityTray', function() {
       assert.ok(stub.notCalled);
     });
 
+    test('not called if non-draggable target', function() {
+      var stub = this.sinon.stub(UtilityTray, 'onTouchStart');
+      var overlay = UtilityTray.overlay;
+      var notification = overlay.querySelector('.notification');
+      var stickyNotification = overlay.querySelector('.fake-notification');
+      var button = overlay.querySelector('button');
+      var li = overlay.querySelector('li');
+
+      [
+        stickyNotification,
+        notification,
+        button,
+        li
+      ].forEach(el => {
+        var fakeEvt = createEvent('touchstart', true, true);
+        fakeEvt.touches = [0];
+        el.dispatchEvent(fakeEvt);
+        assert.ok(stub.notCalled);
+      });
+    });
+
+    test('not called if button is target', function() {
+      var stub = this.sinon.stub(UtilityTray, 'onTouchStart');
+      var notification = UtilityTray.notifications
+        .querySelector('.notification');
+      notification.dispatchEvent(fakeEvt);
+      assert.ok(stub.notCalled);
+    });
```

Comment:
not sure I understand what this test is testing that wasn't tested by the previous test...